upstairs/src/upstairs.rs

```diff
@@ -237,8 +237,8 @@ pub(crate) struct Upstairs {
     /// This is public so that others can clone it to get a controller handle
     pub(crate) control_tx: mpsc::Sender<ControlRequest>,
 
-    /// Stream of post-processed `BlockReq` futures
-    deferred_reqs: DeferredQueue<Option<DeferredBlockReq>>,
+    /// Stream of post-processed `BlockOp` futures
+    deferred_reqs: DeferredQueue<Option<DeferredBlockOp>>,
```

Comment:
About all I can come up with for bikeshedding is calling this `deferred_ops`.
Weak, I know, but you did request comments of that sort

Reply:
Okay, I did a general `req` -> `op` change in 1de0bfaae35bbca5ad225667b086d333bd9c9c6d